device-types/Cisco/FPR3110-NGFW-K9.yaml

```diff
@@ -0,0 +1,54 @@
+---
+manufacturer: Cisco
+model: FirePOWER 3110 Next-Generation Firewall
+slug: cisco-firepower-3110-next-generation-firewall
+part_number: FPR3110-NGFW-K9
+u_height: 1
+is_full_depth: true
+subdevice_role: parent
+comments: '[Cisco Firepower 3100 Series Data Sheet](https://www.cisco.com/c/en/us/products/collateral/security/firewalls/secure-firewall-3100-series-ds.html)'
+console-ports:
+  - name: Console
+    type: rj-45
+  - name: usb
+    type: usb-a
+power-ports:
+  - name: PSU
+    type: iec-60320-c14
+    maximum_draw: 400
```

Comment:
Is the power supply modular?

Reply:
It is hot-swappable, if that's what you mean. It's shipped with 1 PSU, but a second one can be optionally ordered (https://www.cisco.com/c/en/us/td/docs/security/secure-firewall/hardware/3100/fw-3100-install/m-overview.html#concept_pll_mqw_1cb:~:text=FPR3K%2DXNM%2D8X1GF)-,AC%20power%20supply,-Two%20power%20supply)

Reply:
Yes that is correct. Since they are modular (hot-swappable), you should create two items in `module-bays` and then add the applicable PSUs to the module-types folder if they do not already exist